pkgs/desktops/plasma-5/plasma-workspace/0001-startkde.patch

```diff
@@ -48,22 +48,7 @@ index e0f7004..8ac41fd 100644
  }
  
  QStringList allServices(const QLatin1String& prefix)
-@@ -184,14 +184,6 @@ void runEnvironmentScripts()
```

Comment:
TODO: what's up with this

Reply:
This is not required anymore since https://github.com/KDE/plasma-workspace/commit/3c1a34a26dbdf7c123782746881d914cd47b27b6